contributors/devel/development.md

```diff
@@ -247,6 +247,8 @@ _If you have upstream write access_, please refrain from using the GitHub UI for
 creating PRs, because GitHub will create the PR branch inside the main
 repository rather than inside your fork.
 
+When pushing commits to a pull request, don't force push. Force pushing can corrupt your pull request.
```

Comment:
I prefer `Avoid updating existing commits in a pull request. This will hide existing comments on your PR. Instead create separate commits for each push. `